src/app/components/login-button/login-button.component.html

```diff
@@ -0,0 +1 @@
+<span>JETZT ANMELDEN</span>
```

Comment:
no reusable button component. 

generic button component which is just called button to match the following criterias:

1. Input "text" to show as text
2. Definition of type via enum (normal {default}, big)
3. click handler
4. Optional  Input "link" if set shouldn't be rendered as <button> but as <a>

Reply:
Please have a look at the components documentation to see all possibilities:

https://angular.io/guide/component-overview